### PR TITLE
Display booleans and such in table

### LINKED ISSUE #8: Booleans won't be displayed
Solution: apply `str`?

### PR DESCRIPTION
**Pull request by smogg**
closes  #8

### CODE REVIEW

File: src/wilson/dom.cljs [hunk 1]
```diff
@@ -46,13 +46,21 @@
       ((by-key sort-fns) by-key rows)
       ((:default sort-fns) by-key rows)))
 
+(defn parse-td-data
+  [x]
+  (if (not (or (vector? x) (string? x)))
+    (pr-str x)
+    x))
```
Comment (lvh) [inline src/wilson/dom.cljs]: see https://github.com/RackSec/wilson/pull/11#discussion_r61447095

**Comment by lvh** [comment]
LGTM once you flip that if although that's up to you

**Comment by smogg** [comment]
@lvh done, thanks!

File: src/wilson/dom.cljs [hunk 2]
```diff
@@ -46,13 +46,23 @@
       ((by-key sort-fns) by-key rows)
       ((:default sort-fns) by-key rows)))
 
+(defn parse-td-data
+  "Return booleans (and other data other than actual strings or vectors).
```
Comment (lvh) [inline src/wilson/dom.cljs]: This doesn't return a boolean; that's the entire point!

**Pull request merged**